_posts/2023-06-16-sff-students.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Station1 Frontiers Fellowship Students 2023
+---
+<p>
+<img
+src="https://images.squarespace-cdn.com/content/v1/5992e994bebafb41ac4baa11/1512655046354-ZVQXATKKVTLFLLOX2JJB/icon++newwwwww2+%283%29.png"
+title="Station1 Condensed Logo" alt="Station1 Condensed Logo"
+style="float: right; height: 8em; " />
+</p>
+
+MDAnalysis is excited to be participating this year as a host organization for the [Station1 Frontiers Fellowship][sff] program for the first time. We thank Station1 for supporting two students, John Vinh Ong and Karen Bekhazi, who will be working with us to improve and extend MDAnalysis's interoperability. Stay tuned for updates on the project, and keep reading to learn more about John and Karen!
+
+## John Ong
+
+<img
+src="https://jong9559.github.io/assets/img/prof_pic.jpg"
+title="John Ong" alt="John Ong"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Hello everyone, my name is John Ong, and I am one of the Station1 interns at MDAnalysis this summer. I am a soon-to-be third-year undergraduate student at the University of Utah studying mathematics and computer science. Please forgive me for not having introduced myself earlier.
+
+MDAnalysis is core to many professional applications, including treatments for various diseases. The Protein Data Bank (PDB) is an international repository for organic structures and contributes greatly to understanding and creating treatments for diseases. However, data representation techniques for structures on the PDB often differ, despite the same master format (PDBx/mmCIF). Hence, there exists "dialects" within the PDB's files. 
+
+This summer, Karen and I explored the reasons for the existence of these dialects. Through literature and interviews we've carried out with both key members within the PDB and PDB user(s), we've found experimental methods, knowledge of proteins, data processing methods, and user error to be the most common contributors to these dialects. We also studied the motivations for format changes and regulations and the lack thereof within the PDB and its user base. 
+
+All in all, this was a rewarding summer. While we did not achieve as many technical results as we would have liked, studying change has been interesting. I'm honored to have been a mentee of Richard Gowers, Ph.D., and Hugo MacDermott-Opeskin, Ph.D., and to have had the support of MDAnalysis and its community this summer. 
+
+You can find me on [LinkedIn][john-linkedin] and you can find me on [my website][john-website]! 
+
+I hope to be able to continue contributing to this incredible community!
+
+
+## Karen Bekhazi
+
+<img
+src="Karen's Picture Here"
+title="Karen Bekhazi" alt="Karen Bekhazi"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Write something about yourself here!
```

Comment:
Molecular dynamics simulations are capable of generating new trajectory analysis in their own specific output format. This consequently challenges the tools’ further development and widespread use since they are limited to specific formats that can’t be accessed by all users. This summer, Karen and John’s work has mostly focused on the effects the lack of a standardized format have had over molecular dynamics’ scientific and social impact. They evaluated the creation of different Protein Data Bank (PDB) file formats by drawing parallels between them and notions in the field of linguistics, such as dialects and stemmatics. Their research was also supported by further study of both the PDB and MDAnalysis’s stakeholders.

Karen Bekhazi is a rising junior at Smith College. She’s studying Engineering Sciences and Chemistry, with an interest in biochemistry, biomaterials, and the intersection of biophysics and chemistry. Originally from Beirut, Lebanon, she enjoys running, reading, and watching movie commentaries in her free time.

Reply:
@KarenBekhazi you can edit the file from GitHub https://github.com/jennaswa/MDAnalysis.github.io/edit/sff-intro-blog/_posts/2023-06-16-sff-students.md?pr=%2FMDAnalysis%2FMDAnalysis.github.io%2Fpull%2F294 (File and then Edit from the "..." menu).

This will probably create a PR on this PR, which is good. Then someone else can merge your changes in.

Or you use the "suggestions" in your comment:
* [commenting on PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request)
* use a comment as shown in the image below
  ![image](https://github.com/MDAnalysis/MDAnalysis.github.io/assets/237980/83be2856-bce9-456d-9a5a-5205c1802168)

This will give:
```suggestion

Molecular dynamics simulations are capable of generating new trajectory analysis in their own specific output format. This consequently challenges the tools’ further development and widespread use since they are limited to specific formats that can’t be accessed by all users. This summer, Karen and John’s work has mostly focused on the effects the lack of a standardized format have had over molecular dynamics’ scientific and social impact. They evaluated the creation of different Protein Data Bank (PDB) file formats by drawing parallels between them and notions in the field of linguistics, such as dialects and stemmatics. Their research was also supported by further study of both the PDB and MDAnalysis’s stakeholders.

Karen Bekhazi is a rising junior at Smith College. She’s studying Engineering Sciences and Chemistry, with an interest in biochemistry, biomaterials, and the intersection of biophysics and chemistry. Originally from Beirut, Lebanon, she enjoys running, reading, and watching movie commentaries in her free time.
```

Reply:
@KarenBekhazi - Thank you for this summary; I've incorporated it into the blog post (though changed it to first person since it is coming from you!). If you are comfortable doing so, it would be great if you could add a photo (see code from John's section of the code) and/or links to ways to contact you (like GitHub!). You could try out one of the mechanisms @orbeckst suggested above to make the changes